ProcessSlave.php

```diff
@@ -420,8 +420,19 @@ protected function serveStatic(\React\Http\Request $request, HttpResponse $respo
         if ($path === '/') {
             $path = '/index.html';
         }
+        else {
+            $path = str_replace("\\", '/', $path);
+        }
+
+        $basePath = realpath($this->getBridge()->getStaticDirectory());
```

Comment:
That lookup can be cached and doesn't have to happen on each request.

Reply:
Those lookups are cached by PHP. However, we can still save come CPU circle (PHP's normalising the path) by caching it.